src/bp-notifications/bp-notifications-functions.php

```diff
@@ -1587,3 +1587,22 @@ function bb_get_delay_email_notifications_time() {
 function bb_check_delay_email_notification() {
 	return (bool) ( false === bb_enabled_legacy_email_preference() && bb_delay_email_notifications_enabled() );
 }
+
+/**
+ * Function to check current user it offline for sending the push notification.
```

Comment:
Return true or false telling if we can trigger a push notification to the user or not.